operator/identitygc/cell.go

```diff
@@ -81,4 +81,8 @@ func (def Config) Flags(flags *pflag.FlagSet) {
 type SharedConfig struct {
 	// IdentityAllocationMode specifies what mode to use for identity allocation
 	IdentityAllocationMode string
+	// EnableOperatorManageCIDs enables operator to manage CID by
+	// running a CID controller. If enabled, Identity GC cell is
+	// then disabled because CID controller takes care of garbage collection.
+	EnableOperatorManageCIDs bool
```

Comment:
The management of the new option in this cell is correct, but we need to add an operator option to allow the user to set this one. Look at how `IdentityAllocationMode` is initialized in operator/cmd/root.go from the related option in daemonConfig for an example.

Reply:
Hey @pippolo84 thank you for your review!

We'll add this in a later PR, by default this is marked as false. I intentionally not added that here so we don't have to worry about mistakenly enabling this until the full implementation is merged.